<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Lag Tester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_Click">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	font-weight:bold;
	font-size:24px;
	color:#000;
}

*[class=active] {
	background-color:rgb(255, 0, 0);
	font-weight:bold;
	font-size:24px;
	color:#000;
}</string>
      </property>
      <property name="text">
       <string>Click when red!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="label_Count">
      <property name="text">
       <string>Count: N/A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Average">
      <property name="text">
       <string>Average: N/A ms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_State">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
